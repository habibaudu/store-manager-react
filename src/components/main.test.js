import React from 'react';
import { shallow } from 'enzyme';
import Main from './main';

describe('HomePage Component', () => {
  it('should render the Landing Page', () => {
    const component = shallow(<Main />);
    expect(component.exists()).toBe(true);
    expect(component).toMatchSnapshot();
  });
});
