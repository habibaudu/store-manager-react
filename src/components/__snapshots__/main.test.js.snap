// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage Component should render the Landing Page 1`] = `
<Fragment>
  <div
    className="container"
  >
    <div
      className="divide clearfix"
    >
      <section
        className="landing2"
      >
        <img
          src={Object {}}
        />
      </section>
      <section
        className="landing"
      >
        <div
          className="logo_landing"
        >
          <h4>
            Store Manager
          </h4>
        </div>
        <p>
          Welcome to Exclusive shoe's STORE MANAGER app,which aids in effective management of our Store.To continue login with admin provided login credentials
        </p>
        <Link
          className="btn1"
          replace={false}
          to="/login"
        >
          Login
        </Link>
      </section>
    </div>
  </div>
</Fragment>
`;
